### ISSUE TITLE
missing gcsfs in dev env

### ISSUE THREAD

**Issue by raybellwaves** [pytroll/satpy#1306]
I was working with a dev env via https://satpy.readthedocs.io/en/latest/dev_guide/index.html#development-installation
but realized it didn't have the `gcsfs` dependency.

Not sure if a dependency env file could be added (see https://docs.dask.org/en/latest/develop.html#install)

**Comment by djhoese** [comment]
I'm not sure gcsfs would be included in a typical dev env file. It is only used for downloading demo data right now. I think we'd have to have a separate demo environment file or something.

**Comment by raybellwaves** [comment]
Yeah not a bad idea. Maybe an env file that will run all the notebooks in the examples folder

**Comment by raybellwaves** [comment]
Hope you don't mind I renamed the title taking in to consideration of your feedback. I also moved some of my origin of this issue to an issue at pytroll-examples https://github.com/pytroll/pytroll-examples/issues/32

**Comment by djhoese** [comment]
The pytroll-examples repository has a yaml env file that is used by binder. That is probably good enough and i think is described in the readme as an installation option. If not there maybe i put it in my four hour tutorial readme. I'll have to look next week.
